api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.IO.html">System.IO</a></td>
+   <td>
+   Contains types that allow reading and writing to files and data streams, and types that provide basic file and directory support.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.IO.Compression.html">System.IO.Compression</a></td>
+   <td>
+   Contains types that provide basic compression and decompression services for streams. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.IO.IsolatedStorage.html">System.IO.IsolatedStorage</a></td>
+   <td>
+   Contains types that allow the creation and use of isolated stores. Isolated stores let you read and write data that less trusted code cannot access, and prevent the exposure of sensitive information that can be saved elsewhere on the file system. Data is stored in compartments that are isolated by the current user and by the assembly in which the code exists. Roaming profiles can be used in conjunction with isolated storage so that isolated stores travel with the user's profile. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.IO.MemoryMappedFiles.html">System.IO.MemoryMappedFiles</a></td>
+   <td>
+   Contains classes for using a memory-mapped file, which maps the contents of a file to an application’s logical address space. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.IO.Packaging.html">System.IO.Packaging</a></td>
+   <td>
+   Contains types that support storage of multiple data objects in a single container.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.IO.Pipes.html">System.IO.Pipes</a></td>
+   <td>
+   Contains types for interprocess communication through anonymous and/or named pipes.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Linq.html">System.Linq</a></td>
+   <td>
+   Contains types that support queries that use Language-Integrated Query (LINQ).
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Linq.Expressions.html">System.Linq.Expressions</a></td>
+   <td>
+   Contains types that enable language-level code expressions to be represented as objects in the form of expression trees.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Net.html">System.Net</a></td>
+   <td>
+   Provides a simple programming interface for many of the protocols used on networks today. The <a href="http://dotnet.github.io/api/System.Net.WebRequest.html">WebRequest</a> and <a href="http://dotnet.github.io/api/System.Net.WebResponse.html">WebResponse</a> classes form the basis of what are called pluggable protocols, an implementation of network services that enables you to develop applications that use Internet resources without worrying about the specific details of the individual protocols.  
```

Comment:
I'm not sure it makes sense to focus on pluggability here. I don't think that feature is used much, and I think `WebRequest` exists in .Net Core pretty much only for compatibility.